### PR TITLE
Add renew all interaction and message

### PR DESCRIPTION
**Pull request by jonearley**

### CODE REVIEW

File: views/shelf.erb [hunk 1]
```diff
@@ -6,14 +6,33 @@
 
 <% elsif %>
 
+<div class="renew-all-message">
+  <div class="renew-all-message-content">
+    <p><strong>15 items were successfully renewed.</strong></p>
```
Comment (jonearley) [inline views/shelf.erb]: @niquerio could you please replace `15` and `12` with back-end data? Thank you!

File: views/shelf.erb [hunk 2]
```diff
@@ -43,14 +62,10 @@ Renew all could be too slow. Maybe reconsider.
               <option value="30">50 items</option>
```
Comment (erinesullivan) [inline views/shelf.erb]:
```suggestion
              <option value="50">50 items</option>
```

File: views/shelf.erb (hunk 1 above)
Comment (jonearley) [reply views/shelf.erb]: @niquerio and make this message conditional on the renew all message being available from the back-end?

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "webpack-merge": "^5.7.3"
   },
   "scripts": {
-    "watch-css": "node-sass  --source-map true --watch css -o public/bundles",
+    "watch-css": "node-sass --source-map true --watch css -o public/bundles",
```
Comment (erinesullivan) [inline package.json:15]: 🎉

**Pull request merged**